crowdin.yml

```diff
@@ -0,0 +1,20 @@
+project_id: '458576'
+api_token: '94f023f6ed164a9989930454897f55c11c166065d2f072b46adec92462ea35474f4a4f4b84fa9253'
```

Comment:
Is it safe to expose api_token to public? Can you see how Casbin site is doing this?

Reply:
I will add the api to the secrets.